server/motr_common.cc

```diff
@@ -168,6 +168,15 @@ void teardown_motr_op(struct m0_op *op) { teardown_motr_cancel_wait_op(op, 0); }
 void teardown_motr_cancel_wait_op(struct m0_op *op, int sec) {
```

Comment:
At some point, I saw some code in S3 which was doing the same teardown directly, not using this function.  Pls search for op_fini/op_free in code and make sure we won't have memory leak or crash.

Reply:
those functions are now called from teardown only